Apps/APN-UIKit/BuildEnvironment.sample.swift

```diff
@@ -1,8 +1,20 @@
 import Foundation
 
-class BuildEnvironment {
+struct BuildEnvironment {
```

Comment:
With the updated `BuildEnvironment` file, local builds will fail after merging to `main`. Copying the newer version should resolve it easily.